metricflow/engine/metricflow_engine.py

```diff
@@ -294,8 +295,10 @@ def explain_get_dimension_values(  # noqa: D
         """Returns the SQL query for get_dimension_values.
 
         Args:
-            metric_name: Names of metrics that contain the group_by.
+            metric_names: Names of metrics that contain the group_by.
+            metrics: Similar to `metric_names`, but specified via parameter objects.,
```

Comment:
nit: remove comma at the end of this line

Reply:
Removed.